analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -171,7 +171,7 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
             return scm
         }
 
-        fun parseScm(scm: Scm?): VcsInfo {
+        private fun parseScm(scm: Scm?): VcsInfo {
```

Comment:
Just FYI, I deliberately made this public as part of f2b232f40dd0d394e70d72301f8a6cf611275562 even if not required, as I found it strange to have `parseLicenses` and `parseVcsInfo` public but not `parseScm`. But as there seems to be no way to suppress the IDE hint about this, I'm fine with the change.

Reply:
Wasn't aware of that, thanks for pointing it out!